Lesson16/gozde/script.js

```diff
@@ -0,0 +1,263 @@
+// 1. Write a function that takes in a number and checks if it's even or odd using if else statement.
+
+function isEvenOrOdd(number) {
+	const reminder = number % 2;
+	// === - strict equality
+	if (reminder === 0) {
+		return 'Even';
+	} else {
+		return 'Odd';
+	}
+}
+
+console.log('Exercise 1', isEvenOrOdd(9), isEvenOrOdd(2));
+// 2. Write a function that takes age as input. If the age is less than 18, print "Sorry, you are not old enough to vote". If the age is between 18 and 65 (inclusive), print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen".
+function canVote(age) {
+	if (age < 18) {
+		return 'Sorry, you are not old enough to vote';
+	} else if (18 <= age && age <= 65) {
+		return 'You are eligible to vote';
+	} else {
+		return 'You are a senior citizen';
+	}
+}
+
+console.log('Exercise 2', canVote(10), canVote(30), canVote(75));
+
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function compareNumbers() {
+    
+    let firstNumber = parseFloat(prompt("Enter the first number:"));
+
+    let secondNumber = parseFloat(prompt("Enter the second number:"));
+
+    if (firstNumber > secondNumber) {
+        alert(firstNumber + " is greater than " + secondNumber);
+    }
+    else if (firstNumber < secondNumber) {
+        alert(firstNumber + " is less than " + secondNumber);
+    }
+    else {
+        alert(firstNumber + " is equal to " + secondNumber);
+    }
+}
+compareNumbers();
+
+
+// 4. Write a function that takes in a string and checks if the first letter is uppercase using if else statement.
+function isFirstLetterUppercase(stringToCheck){
+const firstLetter = stringToCheck[0];
+  if (firstLetter === firstLetter.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
+
+}
+console.log(isFirstLetterUppercase("Televiision"));
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+
+function checkNumber(num) {
+    
+    if (num > 0) {
+        return "The number is positive.";
+    }
+    else if (num < 0) {
+        return "The number is negative.";
+    }
+    else {
+        return "The number is zero.";
+    }
+}
+
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+
+function isLeapYear(year) {
+    
+    if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
+        return true; 
+    } else {
+        return false; 
+    }
+}
+
+
+// 7. Write a function that takes in a temperature and checks if it's above or below freezing using if else statement.
+
+function checkFreezing(temperature) {
```

Comment:
Same as above about this function, call the function and log the result to the console